qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -1135,33 +1135,90 @@ def add_results(self, record_list: List[ResultRecord]):
             Dict with keys: data, meta
             Data is the ids of the inserted/updated/existing docs
         """
-
         meta = add_metadata_template()
 
-        result_ids = []
-        with self.session_scope() as session:
-            for result in record_list:
+        result_ids = ["placeholder"] * len(record_list)
+
+        results_list = []
+        existing_res = {}
+        new_record_idx, duplicates_idx = [], []
+        conds = []
+
+        conds = [
+            and_(
```

Comment:
Why do you need to use `_or` and `_and` ?
The default for `filter` is `and` of all conditions

Reply:
You are totally right, removed this in the new commit.